Zui README Update (#2652)
brimdata/brim: README.md

# Zui

_The Official Front-End to the [Zed Lake](https://zed.brimdata.io/docs/commands/zed)_

![Zui](https://user-images.githubusercontent.com/3460638/216508967-6936f8fd-0579-4e85-8097-f0e22e6ebc27.png)

[](https://www.brimdata.io/download/)

Zui is a powerful desktop application for exploring and working with data. With features such as:

> - A local Zed Lake instance

Reviewer: ```suggestion
- A local Zed lake instance
```

I'm on-board with @nwt's suggestion above about only capitalizing in the title.

> - Remote Zed Lake connections

Reviewer: ```suggestion
- Remote Zed lake connections
```

- Easy drag-and-drop data ingestion
- Automatic detection of common data formats
- Schema inference during ingestion
- Beautiful, intuitive result views for nested or tabular data
- Named queries with version history
- Query session history to keep track of your work
- Pinnable query fragments to keep your search box uncluttered
- Convenient right-click menus for pivoting and filtering
- Exporting results to a file

## Ready To Try It Out?

**[Download Zui](https://www.brimdata.io/download/)** for your operating system.

Refer to the [installation guide](https://github.com/brimdata/brim/wiki/Installation) and

> [releases notes](https://github.com/brimdata/brim/releases) for more information.

Reviewer: ```suggestion
[release notes](https://github.com/brimdata/brim/releases) for more information.
```

## Powered By Zed

[Zed](https://zed.brimdata.io/docs) offers a fresh and innovative approach to working with data known as "[Super-Structured Data](https://www.brimdata.io/blog/super-structured-data/)".

> Behind Zui is a local Zed Lake instance where you can load your data into pools and use the powerful Zed language to search, analyze, and transform it. Use it to:

Reviewer: ```suggestion
Behind Zui is a local Zed lake instance where you can load your data into pools and use the powerful Zed language to search, analyze, and transform it. Use it to:
```

- Explore deeply nested JSON objects
- View parquet files
- Clean up CSV files by adding type information
- Search heterogeneous NDJSON logs
- Transform data from a legacy database's CDC logs
- Investigate Zeek security logs

Zed has everything you need to make your work easier and more efficient. The [storage layer](https://zed.brimdata.io/docs/formats), [type system](https://zed.brimdata.io/docs/formats/zed), [query language](https://zed.brimdata.io/docs/language/overview), and [zq](https://zed.brimdata.io/docs/commands/zq) command-line tool are just a few of the tools and features that Zed offers to the data community.

## Formerly Known as "Brim"

> For many years, the app was known as Brim, named after the company that created it [Brim Data](https://www.brimdata.io/). In 2023, it was renamed to Zui, a play on "Zed User Interface", to better reflect its connection to the Zed technology that powers it.

Reviewer: ```suggestion
For many years, the app was known as Brim, named after the company [Brim Data](https://www.brimdata.io/) that created it. In 2023, it was renamed to Zui, a play on "Zed User Interface", to better reflect its connection to the Zed technology that powers it.
```

> Zui retains the security-specific features that made Brim popular while expanding it's reach to anyone who has data to work with. For example, you'll still find the customized views, histograms, and correlations relevant to the security domain appearing in Zui via our nacent plugin system. In the future, developers will be able to create custom plugins that make Zui even more effective for their specific needs.

Reviewer: ```suggestion
Zui retains the security-specific features that made Brim popular while expanding its reach to anyone who has data to work with. For example, you'll still find the customized views, histograms, and correlations relevant to the security domain appearing in Zui via our nascent plugin system. In the future, developers will be able to create custom plugins that make Zui even more effective for their specific needs.
```

> Zui retains the security-specific features that made Brim popular while expanding it's reach to anyone who has data to work with. For example, you'll still find the customized views, histograms, and correlations relevant to the security domain appearing in Zui via our nacent plugin system. In the future, developers will be able to create custom plugins that make Zui even more effective for their specific needs.

Reviewer: @nwt: I'm going to drop this suggestion of yours because I made another suggestion in the same line, but I included your it's->its fix in mine as well.
